tests/link_test.toit

```diff
@@ -0,0 +1,52 @@
+// Copyright (C) 2023 Toitware ApS.
```

Comment:
You can use `cwd` to test using absolute paths.

You should test whether relative targets are relative to the link or relative to the current dir.  On Unix, the target is relative to the current directory for hard links, but relative to the link for soft links. Is this also the case for Windows?

Reply:
This doesn't seem to be tested, still.